src/auth-service/controllers/create-user.js

```diff
@@ -281,6 +282,83 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      const { query, body } = req;
+      let { tenant } = query;
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { email, phoneNumber } = req.query;
+      
+       if (!email && !phoneNumber) {
+         return res.status(400).json({ errors: [{ msg: 'Either email or phoneNumber is required' }] });
+    }
+      const hasErrors = !validationResult(req).isEmpty();
+      if (hasErrors) {
+        logObject("hasErrors", hasErrors);
+        let nestedErrors = validationResult(req).errors[0].nestedErrors;
+
+        logger.error(
+          `input validation errors ${JSON.stringify(
+            convertErrorArrayToObject(nestedErrors)
+          )}`
+        );
+
+        return badRequest(
+          res,
```

Comment:
We could minimise the usage of this util for error responses and just send them directly as needed.

Reply:
Thanks @Baalmart

Reply:
Hi @BenjaminSsempala , please ignore this specific feedback since this is the approach we are currently utilising throughout the application. :) 

Please maintain current state and make new commit in case you had made a change based on my earlier feedback.

Thanks again!